src/main/scala/beam/agentsim/agents/ridehail/RideHailManager.scala

```diff
@@ -108,7 +108,11 @@ object RideHailManager {
     def travelDistanceForCustomer(passenger: PersonIdWithActorRef): Double =
       passengerSchedule.legsWithPassenger(passenger).map(_.travelPath.distanceInM).sum
 
-    def toEmbodiedBeamLegsForCustomer(passenger: PersonIdWithActorRef): Vector[EmbodiedBeamLeg] = {
+    //todo we could put rideHailManagerName to TravelProposal itself (or even into RideHailAgentLocation class)
```

Comment:
if this is worth changing - please, create an issue out of this todo, otherwise - let's remove it